scripts/leetcode.js

```diff
@@ -7,6 +7,7 @@ const languages = {
   Java: '.java',
   'C#': '.cs',
   JavaScript: '.js',
+  Javascript: '.js',
```

Comment:
Isn't this line same as the one above?

Reply:
Nope, geeksforgeeks treats it as Java**s**cript while leetcode treats it as Java**S**cript

Reply:
Ahhh I see. What a subtle difference